proto/wippersnapper/signal/v1/signal.proto

```diff
@@ -88,4 +89,16 @@ message SignalResponse {
   oneof payload {
     bool configuration_complete = 1; /** True if a device successfully completed a ConfigurePinRequests message, False otherwise. */
   }
+}
+
+/**
+* PWMRequest represents a broker's request across the PWM sub-topic.
+*/
+message PWMRequest {
+  option (nanopb_msgopt).submsg_callback = true;
+  oneof payload {
+    wippersnapper.pwm.v1.WriteDutyCycle write_duty_cycle = 1;
+    wippersnapper.pwm.v1.WriteFrequency write_frequency  = 2;
+    wippersnapper.pwm.v1.Deinit deinit                   = 3;
+  }
```

Comment:
Do we need to add `WriteDutyCycleMulti` here? I assume it comes across in the same envelope?

Reply:
Good catch, adding..